recipes/qr-code-generator/all/conanfile.py

```diff
@@ -102,7 +102,7 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.libs = [
-            "qrcodegen" if Version(self.version) < "1.7.0" else "qrcodegencpp"
+            "qrcodegen" if Version(self.version) < "1.7.0" else "qrcodegencpp", "qrcodegenc"
         ]
```

Comment:
```suggestion
        ]
        self.cpp_info.components["qrcodegencpp"].set_property("cmake_target_name", "qr-code-generator::qrcodegencpp")
        self.cpp_info.components["qrcodegencpp"].libs = ["qrcodegen" if Version(self.version) < "1.7.0" else "qrcodegencpp"]
        self.cpp_info.components["qrcodegenc"].set_property("cmake_target_name", "qr-code-generator::qrcodegenc")
        self.cpp_info.components["qrcodegenc"].libs = ["qrcodegenc"]
```

I would suggest adding separated components too, so you can link to C++ or C only, by using cmake targets.

Reply:
Ah, that makes sense. I'm pretty new to both Conan and CMake, so I appreciate the feedback!